data/korath/korath variants.txt

```diff
@@ -133,13 +133,15 @@ ship "Rano'erek" "Rano'erek (Shunt)"
 
 ship "Rano'erek" "Rano'erek (Ground Assault)"
```

Comment:
Does this ship still need those empty turrets? If this is a dedicated ground assault vessel then surely the excess turrets would be removed in favour of even more cargo space? (GH isn't letting me put this at the turret lines so placed up here instead)

Reply:
Probably, yes, but trading those in for more space will probably least to balance issues on the crew total and capping balance spectrum.

Reply:
The current changes don't actually change the balance metric, so should be acceptable.